components/match2/wikis/trackmania/match_group_input_custom.lua

```diff
@@ -370,12 +369,11 @@ end
 -- map related functions
 --
 function mapFunctions.getExtraData(map)
-	local overtimes = Array.extractValues(Table.mapValues(mw.text.split(map.overtime or '', ','), tonumber))
 
 	map.extradata = {
 		comment = map.comment,
 		header = map.header,
-		overtime = Table.isNotEmpty(overtimes) and Json.stringify(overtimes) or nil
+		overtime = map.overtime
```

Comment:
if they are to be bools imo use Logic.readBool() here so they get stored with the same value always
currently you could store e.g. 1/yes/true strings or 0/no/false strings
having it already read the bool here is preferable so that the data is the same (true/false bool) everywhere